Ternary Search Tree/Tests/TernarySearchTreeTests.swift

```diff
@@ -10,14 +10,23 @@ import XCTest
 
 class TernarySearchTreeTests: XCTestCase {
     
-    let treeOfStrings = TernarySearchTree<String>()
+
     let testCount = 30
     
     func testCanFindStringInTree() {
         var testStrings: [(key: String, data: String)] = []
+        let treeOfStrings = TernarySearchTree<String>()
+
         for _ in (1...testCount) {
```

Comment:
@pbodsk would it be possible to build the TST once and use that tree to perform your test? Instead of creating it on every test?